stock_picking_cancel_confirm/wizard/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import cancel_confirm
```

Comment:
**In my option, we don't need to patch any of this wizard.**
It was the logic of Odoo core in picking.action_cancel() that do not cancel picking (whether it is a bug or the intention)

BUT, if we need to fix it we should fix at the right place, which is in action_cancel() itself.

Reply:
I propose the following code for **def action_cancel()**

```
    def action_cancel(self):
        if not self.filtered("cancel_confirm"):
            return self.open_cancel_confirm_wizard()
        res = super().action_cancel()
        # Ensure to cancel the picking w/o moves that wasn't cancelled by odoo core
        self.filtered(lambda l: l.state != "cancel" and not l.move_lines).write({"state": "cancel"})
        return res
```

Reply:
@kittiu Thank you for your suggestion. I will change it.